week-4/implement/count.test.js

```diff
@@ -15,3 +15,41 @@
 // And a character char that does not exist within the case-sensitive str,
 // When the function is called with these inputs,
 // Then it should return 0, indicating that no occurrences of the char were found in the case-sensitive str.
+function countChar(str, char) {
+    // Initialize a counter to keep track of occurrences
+    let count = 0;
+
+    // Loop through each character in the string
+    for (let i = 0; i < str.length; i++) {
+        // Check if the current character matches the specified character
+        if (str[i] === char) {
+            // Increment the counter if there is a match
+            count++;
+
+            // Additional logic to handle consecutive occurrences
+            // For example, if char is 'a' and str is 'aaaaa',
+            // we want to count this as one occurrence of 'a'.
+            // So, we can skip consecutive occurrences by incrementing i until a different character is encountered.
```

Comment:
This works but is not what the question asked for I'm afraid - please read the instructions again and tweak it